test/integration/targets/groupby_filter/runme.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
Use `set -eux` here. This will catch use of undefined variables and will make it easier to see what's going on when the test runs (particularly handy when reading Shippable logs).